eng/pipelines/templates/jobs/python-analyze-weekly.yml

```diff
@@ -0,0 +1,86 @@
+parameters:
+  - name: ServiceDirectory
+    type: string
+    default: ''
+  - name: TestMarkArgument
+    type: string
+    default: ''
+  - name: BuildTargetingString
+    type: string
+    default: 'azure-*'
+
+jobs:
+  - job: 'Analyze'
+    timeoutInMinutes: 90
+
+    variables:
+      PythonVersion: '3.8'
+
+    pool:
+      name: azsdk-pool-mms-ubuntu-2004-general
+      vmImage: MMSUbuntu20.04
+
+    steps:
+      - task: UsePythonVersion@0
+        displayName: 'Use Python 3.8'
+        inputs:
+          versionSpec: '3.8'
+      - script: |
+          python -m pip install setuptools==58.3.0
+          python -m pip install -r eng/ci_tools.txt
+        displayName: 'Prep Environment'
+      - task: PythonScript@0
+        displayName: 'Run Pylint Next'
+        continueOnError: true
+        inputs:
+          scriptPath: 'scripts/devops_tasks/dispatch_tox.py'
+          arguments: >-
+            ${{ parameters.BuildTargetingString }}
+            --mark_arg="${{ parameters.TestMarkArgument }}"
+            --service="${{ parameters.ServiceDirectory }}"
+            --toxenv="next-pylint"
+            --disablecov
+            --filter-type="Omit_management"
+        env:
+          GH_TOKEN: $(azuresdk-github-pat)
+
+      - task: PythonScript@0
+        displayName: 'Run MyPy Next'
+        continueOnError: true
+        inputs:
+          scriptPath: 'scripts/devops_tasks/dispatch_tox.py'
+          arguments: >-
+            ${{ parameters.BuildTargetingString }}
+            --mark_arg="${{ parameters.TestMarkArgument }}"
+            --service="${{ parameters.ServiceDirectory }}"
+            --toxenv="next-mypy"
+            --disablecov
+        env:
+          GH_TOKEN: $(azuresdk-github-pat)
+
+      - task: PythonScript@0
+        displayName: 'Run Pyright Next'
+        continueOnError: true
+        inputs:
+          scriptPath: 'scripts/devops_tasks/dispatch_tox.py'
+          arguments: >-
+            ${{ parameters.BuildTargetingString }}
+            --mark_arg="${{ parameters.TestMarkArgument }}"
+            --service="${{ parameters.ServiceDirectory }}"
+            --toxenv="next-pyright"
+            --disablecov
+        env:
+          GH_TOKEN: $(azuresdk-github-pat)
+
+      - task: PythonScript@0
+        displayName: 'Run Ruff'
+        continueOnError: true
+        inputs:
+          scriptPath: 'scripts/devops_tasks/dispatch_tox.py'
+          arguments: >-
+            ${{ parameters.BuildTargetingString }}
+            --mark_arg="${{ parameters.TestMarkArgument }}"
+            --service="${{ parameters.ServiceDirectory }}"
+            --toxenv="ruff"
+            --disablecov
+        env: ${{ parameters.EnvVars }}
```

Comment:
EnvVars vs GH_TOKEN only for the others?

Reply:
Moved to GH_TOKEN for all